web/assets/themes/wp-berlin/front-page.php

```diff
@@ -24,16 +24,29 @@
 
             while ($minutes->have_posts()) :
                 ?>
-                <div class="frontpage-fallback-minutes">
-                    <?php
-                    $minutes->the_post();
+                <div class="main-inner">
```

Comment:
`.main-inner` is the wrapper for the whole page content, this should not be repeated here.